embassy-sync/src/channel.rs

```diff
@@ -29,6 +29,8 @@ use crate::blocking_mutex::raw::RawMutex;
 use crate::blocking_mutex::Mutex;
 use crate::waitqueue::WakerRegistration;
 
+pub mod priority;
```

Comment:
could you put this in `embassy_sync::priority_channel`, to be consistent with `embassy_sync::zerocopy_channel`?